### ISSUE TITLE
Master branch differs from published PyPi version

### ISSUE THREAD

**Issue by seanlindo** [microsoft/presidio-research#83]
`TextAnalyticsWrapper` is not exported in the current published version, `0.1.0`. The following code fails when executing against the published version, but works fine when building from source:

```python
from pathlib                                import Path
from copy                                   import deepcopy
from pprint                                 import pprint
from collections                            import Counter

from presidio_evaluator                     import InputSample
from presidio_evaluator.evaluation          import Evaluator, ModelError
from presidio_evaluator.models              import TextAnalyticsWrapper
from presidio_evaluator.experiment_tracking import get_experiment_tracker
import pandas as pd

def main():    
    print('hello world')

if __name__ == "__main__":
    main()
```

There are no versions or tags, so this will build whatever the current state of the master branch is: `pip install git+https://github.com/microsoft/presidio-research.git`, but you can always fork and tag it yourself as a workaround for this issue.

**Comment by seanlindo** [comment]
You might need to install `plotly` as well. It doesn't seem to be listed in the documentation or required.

**Comment by omri374** [comment]
Thanks @seanlindo. We periodically release a new version, and will make sure all requirements are handled.

**Comment by gustavz** [comment]
The package installed via pip misses this file: presidio_evaluator/data_generator/raw_data/us_driver_license_format.yaml
This leads to the 1_Generate_data.ipynb notebook failing add the driver license provider to the faker.
And even if the yaml is manually copied into the package and the faker can successfully add the provider, generating the data still fails with:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
File [~/.pyenv/versions/3.11.6/envs/piidetection/lib/python3.11/site-packages/faker/generator.py:92](https://file+.vscode-resource.vscode-cdn.net/Users/gustav.vonzitzewitz/workspace/buzok_experiments/PII_detection/presidio/~/.pyenv/versions/3.11.6/envs/piidetection/lib/python3.11/site-packages/faker/generator.py:92), in Generator.get_formatter(self, formatter)
...
AttributeError: 'RecordGenerator' object has no attribute 'us_driver_license'
...
AttributeError: Failed to generate fake data based on template "My driver's license number is {{us_driver_license}}
".You might need to add a new Faker provider! Unknown formatter 'us_driver_license'
```

**Comment by Gojo1729** [comment]
@gustavz I'm still facing the issue with us_driver_license template 
`AttributeError: Failed to generate fake data based on template "My driver's license number is {{us_driver_license}}".You might need to add a new Faker provider! Unknown formatter 'us_driver_license'
`

**Comment by omri374** [comment]
A new version (0.1.2) was released to PyPI and is now aligned with the master branch.

**Comment by omri374** [comment]
Reopening as there's still an issue with the us driver license generation.

**Comment by omri374** [comment]
Fixed in 0.1.3
